docs/data.rst

```diff
@@ -223,6 +223,12 @@ you must take the following into account.
 
   If :meth:`Instance.set_number()` is used to set a value >= 2^53, an ``Error`` will be raised.
 
+.. warning::
+
+  On Windows systems, the string representations of Not a Number and infinity (e.g.,
```

Comment:
I think this should be in the release notes/known issues, separate from the regular documentation. Too many notes like this make reading the docs quite distracting.

Reply:
Ok, this is already in develop and release notes aren't written for Connector yet.
When @rkorte opens a PR with release notes into develop (which will then be cherry-picked into the release branch) I will remove this warning from here and put it in the release notes.